sdk/python/runtime/dagger.json

```diff
@@ -1,5 +1,9 @@
 {
   "name": "python-sdk",
-  "root": "..",
-  "sdk": "go"
+  "sdk": "go",
+  "root-for": [
+    {
+      "source": "."
+    }
```

Comment:
It makes life a little easier for us because it lets us know that we can stop the "find-up" here rather than keep looking to see if we find a `dagger.json` that has a `root-for` this module in some parent dir.

It does look a bit ugly, but in general the goal of these config files is to be human-readable if needed by primarily interacted with via the CLI, which can beautify the presentation of the information in whatever way it wants.

Reply:
Yeah, makes sense.

Reply:
Does this mean `root-for` is a mandatory field?

Reply:
@shykes no it's always automatically configured by the engine right now.

If you do `dagger mod init --name foo --sdk go` then the resulting `dagger.json` will have `root-for` set to `.`.

If you do `dagger mod init -m path/to/foomod --name foo --sdk go` then a `./dagger.json` will be created in your cwd with `root-for` containing `path/to/foomod` and the module's `dagger.json` will be created at `path/to/foomod/dagger.json` without any `root-for`.

I was wondering if we'd want to add an optional explicit flag at some point for configuring the `root-for` of a module at some point, but this behavior seemed like a good default baseline to start for now.